debug/src/index.js

```diff
@@ -1,7 +1,4 @@
-import { initDebug } from './debug';
-import { initDevTools } from './devtools';
-
 if (process.env.NODE_ENV==='development') {
-	initDebug();
-	initDevTools();
+	require('./debug').initDebug();
+	require('./devtools').initDevTools();
```

Comment:
`microbundle` seems to have trouble with `require` statements. They seem to break the `umd` bundle.

/cc @ForsakenHarmony @Andarist

Reply:
Hmm maybe the terser is made to already elim this if it's set for production?

Reply:
Maybe microbundle is atm only focussed on using import/export?

Reply:
**I think** this is a problem with rollup-plugin-commonjs. This whole file is written as module (it has import/export statements) and thus require calls stay untouched - there is no reason (from the plugin's POV) to handle this as `require` is a valid function name, it's not special cased.

In general mixing ESM & CJS doesn't sound quite right, I understand the problem at hand - CJS can be sync & lazy and that would be helpful here.

Reply:
@Andarist Do you think this can be solved by doing a require.resolveWeak here:

https://github.com/developit/preact/pull/1401/files#diff-5092cf2353750c5deb78d54517684aa5R5

Reply:
Not quite, `resoleWeak` is a webpack thing and I assume that you want to support wider range of bundlers etc

Reply:
Well rollup also supports it, but it isn't a solution. As a temporairy fix I added it in the README to brace the import with a `__dev__` check. Another solution could be to or make our own rollup for debug or look for a solution in `microbundle`